frappe/core/doctype/user/user.py

```diff
@@ -891,10 +891,9 @@ def get_active_website_users():
 def get_permission_query_conditions(user):
 	if user=="Administrator":
 		return ""
-
 	else:
 		return """(`tabUser`.name not in ({standard_users}))""".format(
-			standard_users='"' + '", "'.join(STANDARD_USERS) + '"')
+			standard_users="'" + "', '".join(STANDARD_USERS) + "'")
```

Comment:
@Kratos-Freyja You can replace it by `standard_users=", ".join(frappe.db.escape(user) for user in STANDARD_USERS)` so that the API handles quotes and escaping as well.